app/models/book.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Book < ApplicationRecord
+  belongs_to :category
+
+  has_many :author_books, dependent: :destroy
+  has_many :authors, through: :author_books
+
+  validates :name, presence: true
+  validates :price, presence: true
+  validates :description, presence: true
+  validates :quantity, presence: true
+  validates :height, presence: true
+  validates :width, presence: true
+  validates :length, presence: true
+  validates :year_of_publication, presence: true
```

Comment:
u can pass all fields in one line and set them presence: true

validates :description,:name, :title,...., presence: true

Reply:
fixed